examples/simple-lambda.hs

```diff
@@ -40,3 +45,53 @@ code = "\
 \  callback(null, \"some success message\"); \
 \ } \
 \ "
+
+
+role :: Resource
+role = 
+  resource "IAMRole" $
+  IAMRoleProperties $
+  iamRole rolePolicyDocumentObject
+  & iamrPolicies ?~ [ executePolicy ]
+  & iamrRoleName ?~ "MyLambdaBasicExecutionRole"
+  & iamrPath ?~ "/"
+
+  where
+    executePolicy = 
+      iamPolicies
+      (HM.fromList
```

Comment:
is there a better way (a helper?) vs. using the HM.fromList in order to construct a value of type Object?